core/src/main/java/kafka/server/share/SharePartitionManager.java

```diff
@@ -697,7 +696,7 @@ private SharePartitionKey sharePartitionKey(String groupId, TopicIdPartition top
      */
     // Visible for testing.
     long offsetForEarliestTimestamp(TopicIdPartition topicIdPartition) {
-        // TODO: We need to know the isolation level from group configs, for now we are passing Option.empty() for isolationLevel
+        // Isolation level is required when reading from the latest offset hence use Option.empty() for now.
```

Comment:
Isolation level is required => Isolation level is only required ?

Reply:
Done.